app/assets/stylesheets/govuk-component/_breadcrumbs.scss

```diff
@@ -15,4 +17,30 @@
     color: $secondary-text-colour;
     text-decoration: none;
   }
+
+  @include media-down(mobile) {
+    &.collapse-on-mobile li {
+      display: none;
```

Comment:
If no breadcrumbs are marked `is_page_parent` (ie if Navigation Helpers is out-of-date), then breadcrumbs won't show at all on mobile. Is that acceptable?